cli/src/query.rs

```diff
@@ -19,14 +19,15 @@ use crate::{
 #[allow(clippy::too_many_arguments)]
 pub fn query_files_at_paths(
     language: &Language,
-    paths: Vec<String>,
+    paths: &Vec<String>,
     query_path: &Path,
     ordered_captures: bool,
     byte_range: Option<Range<usize>>,
     point_range: Option<Range<Point>>,
     should_test: bool,
     quiet: bool,
     print_time: bool,
+    stdin: bool,
```

Comment:
this is getting a bit unruly, can we refactor this to move these parameters into a `QueryFileOptions` struct, similar to the `ParseFileOptions` for the parse subcommand?